### PR TITLE
Remove Contacts, Personal Details, Degree and Qualifications

### PR DESCRIPTION
**Pull request by alexdesi**
# Context
This removes Contacts, Personal Details, Degree and Qualifications forms, with relevant MVC components

Those components are not relevant anymore. We might want to cherry-pick part of them in the future.

### Link to Trello card

https://trello.com/c/1zjuMU98/995-clean-up-existing-models-controllers-views-from-the-application

### CODE REVIEW

**Comment by alexdesi** [comment]
I am also removing other files (wholly or partially).

File: config/routes.rb [hunk 1]
```diff
@@ -6,15 +6,11 @@
...
-  resources :contact_details, only: %i[new create edit update], path: 'contact-details'
-  resources :degrees, only: %i[new create edit update]
-  resources :qualifications, only: %i[new create edit update]
+  get 'welcome', to: 'welcome#show'
 
   get 'sign-up', to: 'magic_link/sign_up#new', as: :new_sign_up
   post 'sign-up', to: 'magic_link/sign_up#create', as: :sign_up
 
   get 'check-your-answers', to: 'check_your_answers#show'
   get 'application', to: 'tt_applications#show', as: :tt_application
```
Comment (tijmenb) [inline config/routes.rb]:
```suggestion
```

Comment (alexdesi) [reply config/routes.rb]: ops, I missed it! I've searched for others `tt` string in the project, this is the last one.

**Comment by tvararu** [comment]
:100:

**Pull request merged**